src/domains/user/hooks/index.ts

```diff
@@ -0,0 +1,27 @@
+import { useMutation, useQuery } from "@tanstack/react-query";
+import { getMe, updateUser } from "..";
```

Comment:
If you use this in `domains` folder, try to not import from index files. This may lead to circular imports

```suggestion
import { getMe, updateUser } from "../service";
```